RubberduckTests/Mocks/MockParser.cs

```diff
@@ -159,6 +162,15 @@ public static (SynchronousParseCoordinator parser, IRewritingManager rewritingMa
             return (parser, rewritingManager);
         }
 
+        public static IEnumerable<IAnnotation> GetWellKnownAnnotations()
```

Comment:
Why the `Get` prefix?

Reply:
Get prefix removed, after all this is not a getter :)